tap_service_titan/streams.py

```diff
@@ -917,3 +1053,351 @@ class PaymentsStream(ServiceTitanStream):
     def path(self) -> str:
         """Return the API path for the stream."""
         return f"/accounting/v2/tenant/{self._tap.config['tenant_id']}/export/payments"
+
+
+class EmployeesStream(ServiceTitanExportStream):
+    """Define custom stream."""
+
+    name = "employees"
+    primary_keys: t.ClassVar[list[str]] = ["id"]
+    replication_key: str = "modifiedOn"
+    schema = th.PropertiesList(
+        th.Property("id", th.IntegerType),
+        th.Property("userId", th.IntegerType),
+        th.Property("name", th.StringType),
+        th.Property("role", th.StringType),
+        th.Property("roleIds", th.ArrayType(th.IntegerType)),
+        th.Property("businessUnitId", th.IntegerType, required=False),
```

Comment:
@cjohnhanson arent these required parameters unnecessary since the default is False?